src/passes/Asyncify.cpp

```diff
@@ -795,31 +802,36 @@ static bool doesCall(Expression* curr) {
 class AsyncifyBuilder : public Builder {
 public:
   Module& wasm;
+  Type pointerType;
 
-  AsyncifyBuilder(Module& wasm) : Builder(wasm), wasm(wasm) {}
+  AsyncifyBuilder(Module& wasm)
+    : Builder(wasm), wasm(wasm),
+      pointerType(wasm.memories[0]->is64() ? Type::i64 : Type::i32) {}
 
   Expression* makeGetStackPos() {
     return makeLoad(4,
```

Comment:
Doesn't the load size need to be 8 now? This is loading a pointer, which is 8 bytes.

Reply:
Yes, you are right, updated here and elsewhere.